docs/workflows/sync.rst

```diff
@@ -0,0 +1,89 @@
+Synchronize Roles from Galaxy
+=============================
+
+Users can populate their repositories with content from an external source like Galaxy by syncing
+their repository.
+
+Create a Repository
+-------------------
+
+.. literalinclude:: ../_scripts/repo.sh
+   :language: bash
+
+Repository GET Response::
+
+    {
+        "_created": "2019-04-29T15:57:59.763712Z",
+        "_href": "/pulp/api/v3/repositories/1b2b0af1-5588-4b4b-b2f6-cdd3a3e1cd36/",
+        "_latest_version_href": null,
+        "_versions_href": "/pulp/api/v3/repositories/1b2b0af1-5588-4b4b-b2f6-cdd3a3e1cd36/versions/",
+        "description": "",
+        "name": "foo"
+    }
+
+Reference (pulpcore): `Repository API Usage
+<https://docs.pulpproject.org/en/3.0/nightly/restapi.html#tag/repositories>`_
+
+
+Create a Remote
+---------------
+
+Creating a Remote object informs Pulp about an external content source, which most commonly is
+``https://galaxy.ansible.com/`` or another ``pulp_ansible`` instance. In this case, we will be
+syncing all Roles where ``namespace=elastic`` on Galaxy. You can see those roles
```

Comment:
Here you have 'Roles' and 'roles'.

Reply:
You should probably do a search for Role in docs and make sure they're titlecased.

Reply:
fixed in next push, ty.